docs/clusterManagers.html

```diff
@@ -1,307 +1,737 @@
+         -
+        </span>
+        hadr on C and D to see which has more logs.
+       </span>
+      </li>
+      <li>
+       <span class="wikEdList">
+        Issue forced takeover from the one with more logs. Say it is C.  This makes C the primary, with D as principal standby.
+       </span>
+      </li>
+      <li>
+       <span class="wikEdList">
+        Run db2haicu on C and D to re-enable automation.
+       </span>
+       <ul>
+        <li>
+         <span class="wikEdList">
+          <span class="wikEdListTag">
+           At this point, a
+          </span>
+          utomated failover between C and D is enabled.
+         </span>
+        </li>
+       </ul>
+      </li>
+      <li>
+       <span class="wikEdList">
+        <span class="wikEdList">
+         Later on when Datacenter1 comes back.  A and B will likely need to be reinitialized.
+        </span>
+       </span>
+       <ul>
+        <li>
+         <span class="wikEdList">
+          In a full disaster, everything is lost, including the TSA domain.  Even if TSA domain is not lost (eg. in a scenario where only the network to Datacenter1 is lost, everything within Datacenter1 is fine), use db2haicu to destroy the old TSA domain between A and B.  It will be recreated later.
+         </span>
+        </li>
+        <li>
+         Take a new online database backup on C, and restore this backup on A and B.
+        </li>
+        <li>
+         Issue START HADR AS STANDBY on A and B.
+        </li>
+        <li>
+         <span class="wikEdList">
+          <span class="wikEdListTag">
+           I
+          </span>
+          ssue manual graceful takeover on A (after this step, A is the primary, with B as its principal standby, C and D are the auxiliary standbys)
+         </span>
+         .
+        </li>
+        <li>
+         <span class="wikEdList">
+          Run db2haicu on B to create domain containing A and B, defines tie breaker device, the network and DB2 instance resources
+         </span>
+         .
+        </li>
+        <li>
+         <span class="wikEdList">
+          <span class="wikEdListTag">
+           R
+          </span>
+          un db2haicu on A to create the second DB2 instance resource and HADR resource (after this step, HADR failover automation has been enabled in Datacenter1)
+         </span>
+         .
+        </li>
+       </ul>
+      </li>
+     </ul>
+     <h3 dir="ltr">
+      Technote: TSA Setup on two Data Centers in an Environment having DB2 HADR with Multiple Standbys
+     </h3>
+     <p dir="ltr">
+       <!-- This link has been archived. Could not find the related article -->
+      <a href="https://developer.ibm.com/recipes/tutorials/tsa-setup-on-two-data-centers-in-an-environment-having-db2-hadr-with-multiple-standbys/">
+       https://developer.ibm.com/recipes/tutorials/tsa-setup-on-two-data-centers-in-an-environment-having-db2-hadr-with-multiple-standbys/
```

Comment:
This link needs replace.

Reply:
I wasn't able to find a link for  the article. Is there anything else we can replace it with?

Reply:
I think this is close enough, but it's using Pacemaker. 
https://www.ibm.com/docs/en/db2/11.5?topic=hmsd-scenario-deploying-two-sites-multiple-standby-cluster-same-site-failover-automation
For TSA use this one: https://www.idug.org/HigherLogic/System/DownloadDocumentFile.ashx?DocumentFileKey=52df64bc-a067-4fcb-99ff-01781c8836c1